### ISSUE TITLE
Cannot synchronise repositories without namespace

### ISSUE THREAD

**Issue by ddrugeon** [AliyunContainerService/image-syncer#30]
I would like to synchronise an Azure private container registry with one stored in Alibaba container registry located in Beijing. In Azure private container registry, we do not have any namespace in opposite to Alibaba one.

I followed the documentation and configure `image-syncer` with the following json file.

```json
{
    "auth": {
        "xxx.azurecr.io/": {
            "username": "user",
            "password": "***"
        },
        "yyy.cn-beijing.aliyuncs.com": {
            "username": "user2",
            "password": "***"
        }
    },
    "images": {
        "xxx.azurecr.io/test-david:latest": "yyy.cn-beijing.aliyuncs.com/dev/test-david"
    }
}
```

However, image-syncer cannot get the proper authentication. If I debug step by step program, in `func NewRepoURL(url string) (*RepoURL, error)`, the registry cannot be found and default one `registry.hub.docker.com` is set so it cannot authenticate to my source registry.

How can I specify a source registry that do not have any namespace ?

**Comment by hhyasdf** [comment]
Hi~ ddrugeon, I fixed it in release v1.0.5, thanks for your feedback! 
Just reopen it if you still have other questions.

**Comment by ddrugeon** [comment]
Hello,
 
Thanks a lot, it solves my problems. Now I can synchronise my registries from Azure to Aliyun :)
